pkg/cfn/builder/vpc_ipv6.go

```diff
@@ -35,19 +36,60 @@ func (v *IPv6VPCResourceSet) CreateTemplate() (*gfnt.Value, *SubnetDetails, erro
 		EnableDnsSupport:   gfnt.True(),
 		EnableDnsHostnames: gfnt.True(),
 	})
+	v.rs.defineOutput(outputs.ClusterVPC, vpcResourceRef, true, func(val string) error {
+		v.clusterConfig.VPC.ID = val
+		return nil
+	})
 
+	v.rs.newResource(IPv6CIDRBlockKey, &gfnec2.VPCCidrBlock{
+		AmazonProvidedIpv6CidrBlock: gfnt.True(),
+		VpcId:                       gfnt.MakeRef(VPCResourceKey),
+	})
```

Comment:
It seems like this was refactored out of `func addIpv6CidrBlock()` but it has not been deleted from that function, so the CIDR block is created again when this helper func is called below on line 84. Github won't let me comment directly in the helper func because it's not within the scope of lines you've changed!

Reply:
Probably something from the merge conflict. :( Curse you Merges! I have to reread the whole thing dang it. 

Thanks Niki! :)

Reply:
ah it wasn't refactored out but was put there by Jake and when I merged in it didn't detect any of that. I have to restructure things again.

Reply:
lgtm!